pages/onboarding/index.tsx

```diff
@@ -15,8 +67,77 @@ const Onboarding: NextPage = () => {
           </Button>
```

Comment:
i think it would also be best to put the skip button at the end of the form, rather than at the top - you'll see less user engagement if you present them with the option to skip right away